wordpress-mobile/gutenberg-mobile
# Update Gutenberg ref to trunk commit

We mistakenly merged https://github.com/wordpress-mobile/gutenberg-mobile/pull/4066 early before updating its Gutenberg ref to a commit on the `trunk` branch. This updates the Gutenberg ref to a `trunk` commit. 

To test: 
- Verify CI tasks pass. 
- Verify Gutenberg ref is the correct [merge commit](https://github.com/WordPress/gutenberg/pull/35272#event-5529950191): `b4e981167ce49d78ad802faaaf1719be539816c2`

PR submission checklist:
